src/zhinst/toolkit/session.py

```diff
@@ -782,7 +782,8 @@ def connect_device(
                         serial.upper()
                     ]
                     interface = dev_info["INTERFACE"]
-                    if interface == "none":
+                    undefined_interfaces = ("none", "", "unknown")
+                    if interface.lower() in undefined_interfaces:
                         interface = (
                             "1GbE"
                             if "1GbE" in dev_info["INTERFACES"]
```

Comment:
suggestion: I would also make this case insensitive here (use `.lower()` for both) ... just to be sure we do not break things again

Reply:
Changed if I understood the comment correctly.